test.js

```diff
@@ -45,3 +47,29 @@ test('is mobile', function (t) {
 
   t.end()
 })
+
+describe('ua-bruteforce', function () {
+  const limit = 300
+  const checks = {
+    mobile: { result: true },
+    tablet: { result: true, tablet: true },
+    desktop: { result: false }
+  }
+  const testCases = Object.keys(checks).reduce((cases, deviceCategory) => {
+    cases.push(
+      ...new Array(limit).fill().map(() => ({
+        ua: new UserAgent({ deviceCategory }).toString(),
+        ...checks[deviceCategory]
+      }))
+    )
+
+    return cases
+  }, [])
+
+  testCases.forEach(({ ua, result, tablet }) => {
+    test(ua, t => {
+      t.equal(isMobile({ ua, tablet }), result)
```

Comment:
What about testing for false positives?

Reply:
Hmm ... Doesn't the absence of errors in each device category guarantee the correctness of the impl? No test detects that `desktop` is identified as `mobile` device, or vice versa.

Reply:
You're right, the `desktop` category takes care of this